bindings/node/.evergreen/setup_environment.sh

```diff
@@ -21,18 +23,47 @@ mkdir -p ${BIN_DIR}
 export PATH="$BIN_DIR:/opt/mongodbtoolchain/v2/bin:$PATH"
 
 # locate cmake
-. ./.evergreen/find_cmake.sh
+if [ "$OS" == "Windows_NT" ]; then
+  CMAKE=/cygdrive/c/cmake/bin/cmake
+  ADDITIONAL_CMAKE_FLAGS="-Thost=x64 -A x64"
+else
+  chmod u+x ./.evergreen/find-cmake.sh
+  . ./.evergreen/find-cmake.sh
+fi
 
 # this needs to be explicitly exported for the nvm install below
 export NVM_DIR="${NODE_ARTIFACTS_PATH}/nvm"
 mkdir -p ${NVM_DIR}
 
 # install Node.js
-echo "Installing NVM"
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash
-[ -s "${NVM_DIR}/nvm.sh" ] && \. "${NVM_DIR}/nvm.sh"
 echo "Installing Node ${NODE_LTS_NAME}"
-nvm install --lts=${NODE_LTS_NAME}
+if [ "$OS" == "Windows_NT" ]; then
+  NODE_VERSION=$(curl -s -L https://nodejs.org/dist/latest-${NODE_LTS_NAME}/SHASUMS256.txt | grep -oP 'node-v\K\w+.\w+.\w+' | head -1)
```

Comment:
nit: what is this doing? I assume it is right, but a comment might be neat.

Reply:
added a comment